integration-cli/docker_utils.go

```diff
@@ -1353,3 +1353,33 @@ func createTmpFile(c *check.C, content string) string {
 
 	return filename
 }
+
+func buildImageArgs(args []string, name, dockerfile string, useCache bool) (string, error) {
+	id, _, err := buildImageWithOutArgs(args, name, dockerfile, useCache)
+	return id, err
+}
+
+func buildImageWithOutArgs(args []string, name, dockerfile string, useCache bool) (string, string, error) {
```

Comment:
there are two things that puzzle me about this function:

1. `Without` is a word, and doesn't require the capitalized o.
2. The function suffix is without args and the first parameter is called args.

:confounded: